Jamulus.pro

```diff
@@ -301,6 +301,16 @@ DISTFILES_OBOE += libs/oboe/AUTHORS \
         android/AndroidManifest.xml \
         android/sound.h \
         android/sound.cpp
+
+    isEmpty(PREFIX) {
+        PREFIX = /usr/local
+    }
+    isEmpty(BINDIR) {
+        BINDIR = bin
+    }
+    BINDIR = $$absolute_path($$BINDIR, $$PREFIX)
```

Comment:
I would move this to the top of unix conditional. It will be useful when we decide to install other files like `.desktop` entry or icon.

Reply:
Oh, these are actually available in the repo and AUR package installs them:

https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=jamulus#n26

```qmake
isEmpty(DATADIR) {
    DATADIR = share
}
DATADIR = $$absolute_path($$DATADIR, $$PREFIX)
ICONDIR = $$DATADIR/icons/hicolor/48x48/apps # https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#install_icons
DESKTOPDIR = $$DATADIR/applications
```

We should probably do that too.